Makefile

```diff
@@ -297,7 +298,7 @@ dist/agent-windows-installer.exe: dist/agent-windows-amd64.exe
 	cp LICENSE ./packaging/windows
 ifeq ($(BUILD_IN_CONTAINER),true)
 	docker build -t windows_installer ./packaging/windows
-	docker run --rm -t -v "${PWD}:/home" windows_installer
+	docker run --rm -t -v "${PWD}:/home" -e VERSION=${RELEASE_TAG} windows_installer
```

Comment:
Hmm, can't we use ${IMAGE_TAG} here?

Reply:
They are equivalent here, we could likely clean up to only use one of them.

Reply:
Though there is a situation where you can manually override each one and they would be different, not even sure if that's a valid case?

Reply:
Yeah, I'm not sure that's something we'd want to explicitly support